Src/FluentAssertions/Equivalency/Matching/MustMatchByNameRule.cs

```diff
@@ -35,8 +35,7 @@ public IMember Match(IMember expectedMember, object subject, INode parent, IEqui
             Execute.Assertion.FailWith(
                 $"Expectation has {expectedMember.Description} that the other object does not have.");
         }
-
-        if (options.IgnoreNonBrowsableOnSubject && !subjectMember.IsBrowsable)
+        else if (options.IgnoreNonBrowsableOnSubject && !subjectMember.IsBrowsable)
```

Comment:
🔧 I also expected an additional (empty) `else` here.

Reply:
Not sure what you're saying here?
Something related to AV1537? (Finish every `if`-`else`-`if` statement with an `else` clause)

Reply:
Like this?

```cs
if (subjectMember is null)
{
    Execute.Assertion.FailWith(
        $"Expectation has {expectedMember.Description} that the other object does not have.");
}
else if (options.IgnoreNonBrowsableOnSubject && !subjectMember.IsBrowsable)
{
    Execute.Assertion.FailWith(
        $"Expectation has {expectedMember.Description} that is non-browsable in the other object, and non-browsable " +
        "members on the subject are ignored with the current configuration");
}
else
{
    // Everything is fine
}
```

Reply:
Yep. To make it explicit that you didn't forget about that scenario.